### ISSUE TITLE
Add auto test for web app

### ISSUE THREAD

**Issue by xicheng87** [lucien386/MusicXML2ABCNotationWebApp#19]
As we build up the application, we need to add rigorous tests to prevent regression. I did some quick look on how to test a flask application, you may want to follow below:

https://flask.palletsprojects.com/en/1.1.x/testing/
https://www.patricksoftwareblog.com/testing-a-flask-application-using-pytest/

I haven't looked into this with more details, but I'd think we need to verify some end-to-end behavior, including:

1) Add the music XML file to the text box directly and click "submit"
2) Add the actual file and then click submit. 
3) Add an invalid XML and then click submit.

This list is not exhaustive, but at least we need to have some basic testing for the above cases. 

Basically what happened underneath when you click through a web app is the browser (client) sends HTTP requests to the server, which could and should be tested by python in this case 

Mind looking into this once you finish the fron-end polishing?

**Comment by lucien386** [comment]
I have finished the test on uploading files, still working on submitting texts

**Comment by lucien386** [comment]
finished
